src/utils.ts

```diff
@@ -315,10 +328,16 @@ const getEstimateForStoryType = (story: PivotalStory) => {
  * @param  {StoryResponse} story
  * @returns string
  */
-export const getPrDescription = (body: string = '', story: PivotalStory): string => {
+export const getPrDescription = (body: string = '', story: PivotalStory, reviews: PivotalStoryReview[]): string => {
   const { url, id, story_type, labels, description, name } = story;
   const labelsArr = (labels as Label[]).map((label: { name: string }) => label.name).join(', ');
 
+  // Mention Me Design Review Type IDs
+  const mmDesignReviewTypes = [4013928, 4884434];
```

Comment:
I'd be very careful here.

This is a fork of a public repo - and generally, it's best, where possible to always make changes which can be merged upstream so others can benefit.

This is highly specific to Mention Me. This means our fork is going to be permanent - and we probably don't really want to do that. It'll make it hard to bring in changes from upstream (and we don't get to contribute it back this way either).

Can we think of a better way to do this? Could we make it generic? (See comments further down too - you might have painted yourself into a corner which I can help you out of!)